src/rulesets/oas/functions/__tests__/oasOpIdUnique.test.ts

```diff
@@ -6,7 +6,6 @@ import oasOpIdUnique from '../oasOpIdUnique';
 
 describe('oasOpIdUnique', () => {
   const s = new Spectral();
-  s.registerFormat('oas2', () => true);
```

Comment:
@philsturgeon I've dropped this as this is supposed to not be tied to oas2. Does that make sense? Another option would be to revert that change and add a new test targeting oas3.

Reply:
I think a lot of thse registerFormat's were erroneous and this is fine.